app/views/admin/add_kudos/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2> Add Kudos Points to all employees</h2>
```

Comment:
As I get it - clicking on 'Add Kudos' button should redirect to the page with the form, but on the other hand, we should have a way to check the employees points values somehow.